src/providers/ldap/sdap_async_connection.c

```diff
@@ -1832,6 +1834,11 @@ static void sdap_cli_auth_step(struct tevent_req *req)
      */
     now = time(NULL);
     expire_timeout = dp_opt_get_int(state->opts->basic, SDAP_EXPIRE_TIMEOUT);
+    expire_offset = dp_opt_get_int(state->opts->basic, SDAP_EXPIRE_OFFSET);
+    if (expire_offset > 0) {
+        expire_timeout += sss_rand() % (expire_offset + 1);
+    }
```

Comment:
Would not it make sense to log error if value is negative?

Reply:
Error is now logged in case of negative value